jablib/src/main/java/org/jabref/logic/bibtex/comparator/FieldValueValidityComparator.java

```diff
@@ -0,0 +1,17 @@
+package org.jabref.logic.bibtex.comparator;
+
+import java.util.Comparator;
+
+public class FieldValueValidityComparator implements Comparator<String> {
+    /**
+     * Compares the validity or appropriateness of two field values.
+     *
+     * @param leftValue  value from the fetcher (or candidate)
+     * @param rightValue value from the library (or existing record)
+     * @return 1 if left is better, 0 if undetermined or equal, -1 if left is worse
```

Comment:
Use an `enum` for better readabiality. int offers more values which can be confusing for other developers.

--> strong typing!

We accept that not `Comparator<String>` can be used.

Futhermore, the semantics of `Comparator` is more on the values itself not on the semantics of the values :)

E.g. 2500 is smaller than 2025 in your code :)